ursgal/uparams.py

```diff
@@ -1,4 +1,137 @@
 ursgal_params = {
+    'reporter_ion_tolerance': {
```

Comment:
Possibly a dumb question, but why is that not the same as frag_mass_tolerance?
I know, Orbitraps have different accuracies depending on the m/z, but your engine doesn't use any other frag ions, right? So the frag_mass_tolerance can be set accordingly, before executing the engine!?

Reply:
I did that because I'd set the accuracy about one order of magnitude lower than for matching fragments ions, but you are right, since I don't use other fragments that differentiation is not really necessary, I'm gonna change that